target/linux/ipq40xx/image/Makefile

```diff
@@ -231,6 +231,19 @@ define Device/engenius_eap1300
 endef
 TARGET_DEVICES += engenius_eap1300
 
+define Device/engenius_eap2200
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := EnGenius
+	DEVICE_MODEL := EAP2200
+	BOARD_NAME := eap2200
+	SOC := qcom-ipq4019
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	DEVICE_PACKAGES := ath10k-firmware-qca9888-ct ipq-wifi-engenius_eap2200
```

Comment:
Due to the introduction of 1ac627024de916eb157659caf49f93fa89d3b19a, it's now possible to prevent OOMs when all three cards are sending out data. But for this, you'll need to switch over to "kmod-ath10k-ct-smallbuffers" by appending `-kmod-ath10k-ct kmod-ath10k-ct-smallbuffers` . The question is, whenever you want it or not.

Reply:
Thanks for comment. I append the packages.